activerecord/test/cases/adapter_test.rb

```diff
@@ -23,7 +23,8 @@ def test_update_prepared_statement
     end
```

Comment:
4 test cases on this file should test `tables` and `table_exists?`, so we can not change methods.